sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConfigurationClientJavaDocCodeSnippets.java

```diff
@@ -275,45 +276,57 @@ public void deleteSetting() {
      */
     public void listSettings() {
         ConfigurationClient configurationClient = createSyncConfigurationClient();
-        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettings#SettingSelector
-        for (ConfigurationSetting setting : configurationClient.listSettings(new SettingSelector()
-            .keys("prodDBConnection"))) {
+        // BEGIN: com.azure.data.applicationconfig.configurationclient.listSettings#settingSelector
+        SettingSelector settingSelector = new SettingSelector().keys("prodDBConnection");
+        PagedIterable<ConfigurationSetting> csStream =  configurationClient.listSettings(settingSelector);
+        csStream.forEach(setting -> {
```

Comment:
Don't bother creating the intermediate csStream variable, just do `configurationClient.listSettings(settingSelector).forEach`

Reply:
Do this in all cases in the snippets below too.